deb/jjs-oneclick

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+if [ "$(whoami)" != root ]
+then
+    echo 'jjs-oneclick must be run as root!'
+    exit 1
+fi
+which psql >/dev/null || exit $?
+
+echo '# Setting up config'
+su -c 'jjs-setup --data-dir /var/lib/jjs --install-dir /usr/share/jjs --setup-config' -s "$SHELL" jjs
+
+echo '# Setting up problems'
+(
+    cd /usr/share/jjs/example-problems
+    su -c '
+    for i in *
+    do
+        echo "* $i"
+        mkdir /var/lib/jjs/var/problems/"$i"
+        JJS_PATH=/var/lib/jjs jjs-ppc compile --pkg "$i" --out /var/lib/jjs/var/problems/"$i"
+    done
+    ' -s "$SHELL" jjs
+)
+
+echo '# Setting up database'
```

Comment:
Use jjs-setup instead

Reply:
Not possible until it supports partial db setup